proto/info/info.proto

```diff
@@ -836,22 +836,56 @@ message DevicePort {
   WirelessStatus wireless_status = 33; // defined for cellular/WiFi ports only
   uint32 mtu = 34; // MTU of the port
 
-  // Local modification status for this port.
-  // If modified, the configuration shown reflects local changes and
-  // represents the actual state of the port, not just a copy of the
-  // controller-submitted configuration.
-  LocalModification local_modification = 40;
-}
-
-// Information about locally made configuration changes for this port.
-message LocalModification {
-  // True if this port’s configuration has been locally modified
-  // (e.g., via the Local Profile Server) and differs from the
-  // controller-provided configuration.
-  bool modified = 1;
-
-  // Timestamp of when the last local change for this port was submitted by LPS.
-  google.protobuf.Timestamp submitted_at = 2;
+  // Identifies the source of this network port configuration.
+  PortConfigSource config_source = 40;
+}
+
+// NetworkConfigOrigin enumerates all possible origins of a network port configuration.
+enum NetworkConfigOrigin {
+  // Unknown or unset origin.
+  NETWORK_CONFIG_ORIGIN_UNSPECIFIED = 0;
+  // Config received from the controller.
+  NETWORK_CONFIG_ORIGIN_CONTROLLER = 1;
+  // Initial device config embedded in the EVE installer, signed by the controller.
+  NETWORK_CONFIG_ORIGIN_BOOTSTRAP = 2;
+  // Manually created JSON config injected into the installer.
+  NETWORK_CONFIG_ORIGIN_OVERRIDE = 3;
+  // Fallback configuration automatically generated to enable DHCP on all Ethernet ports.
+  // Used when no network config is available or when "network.fallback.any.eth" is enabled
+  // and none of the existing configs provide controller connectivity.
+  NETWORK_CONFIG_ORIGIN_LASTRESORT = 4;
+  // Config entered via the terminal UI (TUI).
+  NETWORK_CONFIG_ORIGIN_TUI = 5;
+  // Configuration signed by the controller and delivered through the
+  // Local Operator Console (LOC) in an air-gapped environment.
+  NETWORK_CONFIG_ORIGIN_LOC = 6;
+  // Config changes made locally through the Local Profile Server (LPS).
+  NETWORK_CONFIG_ORIGIN_LPS = 7;
+}
+
+// PortConfigSource describes the origin of the configuration used for a network port.
+// It helps distinguish between controller-provided, locally-modified, or initial configs.
+message PortConfigSource {
+  // Indicates where EVE obtained the network config.
+  NetworkConfigOrigin origin = 1;
+
+  // Timestamp when EVE received or generated this configuration.
+  //
+  // Notes:
+  // - DevicePortStatus.timePriority contains the reception timestamp for controller-sourced
+  //   configs. For other sources, this value may not reflect the actual time the configuration
+  //   was generated or received.
```

Comment:
"received" seems a bit underspecified since unchanged configuration is "received" from the controller every 60 seconds by default, and there can be a change to the configuration from the controller at time X but that has no change to this port.

So is it received_change_at or received_at with the comment saying that it is the most recent change to this port?